pkg/handler/project/project_test.go

```diff
@@ -0,0 +1,74 @@
+package project
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/gin-gonic/gin"
+	_ "github.com/lib/pq"
+	"github.com/stretchr/testify/require"
+
+	"github.com/dwarvesf/fortress-api/pkg/config"
+	"github.com/dwarvesf/fortress-api/pkg/logger"
+	"github.com/dwarvesf/fortress-api/pkg/service"
+	"github.com/dwarvesf/fortress-api/pkg/store"
+	"github.com/dwarvesf/fortress-api/pkg/view"
+)
+
+func TestHandler_UpdateProjectStatus(t *testing.T) {
+	// load env and test data
+	cfg := config.LoadTestConfig()
+	loggerMock := logger.NewLogrusLogger()
+	serviceMock := service.New(&cfg)
+	storeMock := store.New(&cfg)
+
+	tests := []struct {
+		name             string
+		wantCode         int
+		wantErr          error
+		wantResponsePath string
+	}{
+		{
+			name:             "ok_update_project_status",
+			wantCode:         200,
+			wantErr:          nil,
+			wantResponsePath: "testdata/update_project_status/200.json",
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			w := httptest.NewRecorder()
+			ctx, _ := gin.CreateTestContext(w)
+			bodyReader := strings.NewReader(`
+			{
+				"status":"active"
+			}
+			`)
```

Comment:
the body must be get from `tests`, not declared in running phase

Reply:
example `https://github.com/dwarvesf/fortress-api/pull/29/files#diff-6d6a317eb1b0c03494708990c7cf6d81dbaad29a27adeed3b7d717b7cd6419dc`

Reply:
thank you, it's done